src/pages/using-spark/guides/writing-meaningful-web-content-with-semantic-html.mdx

```diff
@@ -0,0 +1,250 @@
+---
+title: Writing Meaningful Web Content With Semantic HTML
+---
+
+import { SprkHeading } from '@sparkdesignsystem/spark-react';
+
+<span
+  id="writing-meaningful-web-content-with-semantic-html"
+  class="docs-b-Link--anchor"
+></span>
+<SprkHeading
+  element="h1"
+  additionalClasses="sprk-u-mbl sprk-u-Measure"
+  isPageTitle
+>
+  Writing Meaningful Web Content with Semantic HTML
+</SprkHeading>
+
+Typically, in a word document, writers bold or italicize text to add emphasis to the copy.
+
+When producing web content, **HTML** (Hyper Text Markup Language) **elements** define the structure, purpose and intended meaning of a page's web content.
+
+<SprkHeading
+  element="h3"
+  additionalClasses="sprk-u-mbm sprk-u-Measure"
+  variant="displayThree"
+>
+  Main Takeaways
+</SprkHeading>
+
+- Use HTML elements to define the structure and meaning of a web page's content.
+- Know the purpose of each element to understand when to use it and to ensure the content can be access by assistive technologies.
+- Use the correct element to ensure the content communicates the intended meaning.
+
+Here are some benefits of using semantic HTML
+
+- **It's easier to develop**. It gives content structure making it easier to understand.
+- **It's better for mobile experiences**. Semantic HTML is lighter in file size, and easier to make responsive.
+- **It's good for SEO**. It makes documents more findable, search engines give more importance to keywords inside the content structure.
```

Comment:
These bullets are very similar to the bullets on [this page](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML). I think it's all useful content, but I think we should rewrite this in a more unique way.

Reply:
That was my inspiration, I rewrote them - let me know your thoughts.
- It's easier for engineers to understand and to develop with.
- Create faster, responsive, and better mobile user experiences. It's easier to make responsive, have reduced file size, and load faster.
- Keywords inside documents make them easier to find. Search engines give more importance to those that have them.

Reply:
Much better! Thank you!